pkg/utils/build/quay.go

```diff
@@ -37,9 +37,9 @@ func IsImageAnnotationPresent(annotations map[string]string) bool {
 	return image_annotation_str != ""
 }
 
-func ImageAnnotationGenerateValueIsNotFailed(annotations map[string]string) bool {
-	generate_value_str := annotations["image.redhat.com/generate"]
-	return generate_value_str != "failed"
+func ImageRepoCreationIsNotFailed(annotations map[string]string) bool {
```

Comment:
What about `ImageRepoCreationSucceeded`?

Reply:
Since we are checking whether the image repo creation failed in this function. Would rather like to keep it as `ImageRepoCreationFailed` . WDYT?

Reply:
Agree. Updated now.